src/backend/distributed/deparser/ruleutils_11.c

```diff
@@ -5163,27 +5163,6 @@ get_rule_expr(Node *node, deparse_context *context,
 			}
 			break;
 
-		case T_RelabelType:
-			{
-				RelabelType *relabel = (RelabelType *) node;
-				Node	   *arg = (Node *) relabel->arg;
-
-				if (relabel->relabelformat == COERCE_IMPLICIT_CAST &&
-					!showimplicit)
-				{
-					/* don't show the implicit cast */
-					get_rule_expr_paren(arg, context, false, node);
-				}
-				else
-				{
-					get_coercion_expr(arg, context,
-									  relabel->resulttype,
```

Comment:
hmz, this logic seems sort of important. RelabelType seems to occassionally mean "COLLATE" and in other cases a cast like "::varchar(3)".